hermes/probe/delete/delete.go

```diff
@@ -83,29 +87,31 @@ func DeleteFile(ctx context.Context, fileID int32, target *probe.Target, client
 			status = metrics.ProbeFailed
 		}
 
-		target.LatencyMetrics.APICallLatency[metrics.APIDeleteFile][status].Metric("hermes_api_latency_seconds").AddFloat64(time.Now().Sub(start).Seconds())
+		target.LatencyMetrics.APICallLatency[metrics.APIDeleteFile][status].Metric(apiLatency).AddFloat64(time.Now().Sub(start).Seconds())
```

Comment:
This update should be factored out as it repeats many times throughout the code.

Reply:
I think this is also applicable on other files too, so I've opened issue #77 so this refactoring is done consistently throughout all files (it might also be necessary to tweak) metrics.go (?) in the process so lets do this in a different PR.

@evanSpendlove In the meantime, could you please add a TODO comment where relevant and reference #77?

Reply:
I've added a TODO.